leapp/messaging/answerstore.py

```diff
@@ -7,6 +7,14 @@
 from leapp.utils.audit import create_audit_entry
 
 
+def _comment_out(key, text):
+    """Returns a commented-out string. If any newlines are present it properly deals with them"""
+    split_by_newline = [s for s in text.split('\n') if s.strip()]
+    res = '# {:<20}{}\n{}'.format('{}:'.format(key), split_by_newline[0],
+                                  '\n'.join(['# {}{}'.format(' ' * 20, l) for l in split_by_newline[1:]]))
+    return res
```

Comment:
@fernflower  I could not help myself, but somehow I like the recursion in this function :) no need to do it in this way of course
```suggestion
def _comment_out(key, text, key_separator=':'):
    """Returns a commented-out string. If any newlines are present it properly deals with them"""
    if not text.strip():
        return ''
    split_by_newline = text.split('\n', 1)
    if len(split_by_newline) == 1:
        return '# {:<20}{}\n'.format('{}{}'.format(key, key_separator), split_by_newline[0])
    else:
        return '# {:<20}{}\n{}'.format('{}{}'.format(key, key_separator), split_by_newline[0], _comment_out('', split_by_newline[1], ''))
```

Reply:
Making it recursive is a nice idea, thanks! Took a bit different approach though.